api/swagger-spec/swagger.yml

```diff
@@ -291,6 +332,11 @@ parameters:
     in: path
     description: A universal Id to represent a sepcific request or report
     required: true
+securityDefinitions:
+  basicAuth:
+    type: basic
+security:
+  - basicAuth: []
```

Comment:
removing this should avoid auth for rest API paths that doesn't have security mentioned